boms/common-ee/pom.xml

```diff
@@ -4808,6 +4808,37 @@
                 <version>${version.org.reactivestreams}</version>
             </dependency>
 
+            <dependency>
```

Comment:
Should non-EE artifacts be added to boms/common-ee or boms/common-expansion?

Reply:
@scottmarlow Non-EE artifacts can go in common-ee. But a realize now these particular ones should be in common-expansion not common-ee.

The 'ee' refers to the 'wildfly-ee' feature pack, which is much more than EE; it's just the name we chose for the 'base' FP back in the WF 19/20 days we when split out the 'expansion' stuff. Down stream the wildfly-ee feature pack becomes the 'eap' feature pack used for base EAP (which is much more the Jakarta EE), while the 'wildfly' feature pack becomes the 'eap-xp' FP used for EAP XP. Upstream I refer to things specific to the 'wildfly' feature pack (boms, testsuite modules etc) as 'expansion'. But the feature pack itself is just 'wildfly' because in community 99%+ of people only deal with that FP directly, with wildfly-ee as an internal detail.

The boms/xxx-ee modules are meant to relate to wildfly-ee, while boms/xxx-expansion relate to the 'wildfly' feature pack. These deps are only relevant to the 'wildfly' FP so I will move them to boms/common-expansion.

Reply:
@scottmarlow I've moved these to common-expansion. While doing it I realized one of them wasn't needed at all so I removed that.

When doing that I also rebased on current main and took the opportunity to squash down from three commits to two.